tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -689,3 +690,33 @@ def test_positive_overview_label_name(form_data, session):
             assert results['overview'][key] == value
         session.virtwho_configure.delete(name)
         assert not session.virtwho_configure.search(name)
+
+
+def test_positive_last_checkin_status(form_data, session):
+    """Verify the Last Checkin status on Content Hosts Page.
+
+    :id: 3931e10e-8adc-4ca4-8963-20572b2f99bf
+
+    :expectedresults: The Last Checkin time on Content Hosts Page is UTC time
+
+    :BZ: 1652323
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        values = session.virtwho_configure.read(name, widget_names='deploy.command')
+        command = values['deploy']['command']
+        hypervisor_name, guest_name = deploy_configure_by_command(command, debug=True)
+        time_now = datetime.utcnow()
+        assert session.virtwho_configure.search(name)[0]['Status'] == 'ok'
+        checkin_time = session.contenthost.search(hypervisor_name)[0]['Last Checkin']
+        # 10 mins margin to check the Last Checkin time
+        assert datetime.strptime(checkin_time, "%b %d, %I:%M %p").replace(
```

Comment:
The expression should be in `abs()` so values like "minus ten minutes" are handled correctly.

Reply:
ok,thanks